mobile/native/src/ln_dlc/node.rs

```diff
@@ -411,6 +412,11 @@ impl node::Storage for NodeStorage {
         db::get_channel(user_channel_id)
     }
 
+    fn get_channel_by_fake_scid(&self, _fake_scid: FakeScid) -> Result<Option<Channel>> {
+        // The app does not support loading channels by fake scid
+        unimplemented!("The app does not support loading channels by fake scid")
+    }
```

Comment:
👍I think this is okay, although it reminds me of how much I hate the `Storage` trait I added.